src/pocket/PocketECSCodePipeline.ts

```diff
@@ -17,6 +17,14 @@ export interface PocketECSCodePipelineProps {
     appSpecPath?: string;
     taskDefPath?: string;
   };
+  /** Optional stages to run before the deploy stage.
+   * For CodeBuild actions, ensure that the project name starts with `prefix`.
+   */
+  preDeployStages?: codepipeline.CodepipelineStage[];
+  /** Optional stages to run after the deploy stage.
+   * For CodeBuild actions, ensure that the project name starts with `prefix`.
+   */
+  postDeployStages?: codepipeline.CodepipelineStage[];
```

Comment:
Note: I added both `preDeployStages` and `postDeployStages`. When I was iterating on the custom stage it was convenient to run it first, to not have to wait 5 minutes for the deployment to finish. I can imagine `preDeployStages` would also be useful to run automated tests.